drivers/isl29125/include/isl29125_params.h

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (C) 2017 Freie Universität Berlin
+ * Copyright (C) 2018 HAW-Hamburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     drivers_isl29125
+ *
+ * @{
+ * @file
+ * @brief       Default configuration for ISL29125 devices
+ *
+ * @author      Hauke Petersen <hauke.petersen@fu-berlin.de>
+ * @author      Michel Rottleuthner <michel.rottleuthner@haw-hamburg.de>
+ */
+
+#ifndef ISL29125_PARAMS_H
+#define ISL29125_PARAMS_H
+
+#include "board.h"
+#include "isl29125.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief   Set default configuration parameters
+ * @{
+ */
+#ifndef ISL29125_PARAM_I2C
+#define ISL29125_PARAM_I2C              I2C_DEV(0)
```

Comment:
Probably should == TEST_ISL29125_I2C

Reply:
I removed the parameters from the tests Makefile so there is only the one definition in this file, which can still be overwritten.